src/main/java/de/vill/model/constraint/MultiOrConstraint.java

```diff
@@ -0,0 +1,97 @@
+package de.vill.model.constraint;
```

Comment:
We should probably remove Regular OrConstraint altogether (not yet necessarily). Or does keeping it have any benefit?

Reply:
I agree, that removing it complete might be a good idea. However, I did not do it yet, because I was unsure if there are others depending on it.